package.json

```diff
@@ -37,6 +38,8 @@
     "merge-stream": "1.0.1",
     "moment": "2.19.3",
     "pmm": "2.0.0",
+    "postcss": "^7.0.5",
```

Comment:
We don't want the caret, so it bumps up to the lastest major version on any package

` "postcss": "7.0.5",`

Reply:
Yep.